python_program/app.py

```diff
@@ -0,0 +1,82 @@
+import functools
+import io
+import json
+import os
+import pathlib
+import uuid
+import zipfile
+
+import flask
+from flask import Flask, request, flash, redirect
+from flask_cors import CORS
+
+from main import generate_automated_walk_table, create_arg_parser
+
+app = Flask(__name__)
+cors = CORS(app, resources={r"/*": {"origins": "*"}})
+
+ALLOWED_EXTENSIONS = set(['gpx'])
+
+
+def allowed_file(filename):
+    return '.' in filename and filename.rsplit('.', 1)[1].lower() in ALLOWED_EXTENSIONS
+
+
+@app.route('/create', methods=['POST'])
+def create_map():
+    download_id = str(uuid.uuid4().hex)
+    os.mkdir('./output/' + download_id)
+
+    if 'file' not in request.files:
+        flash('No file part')
+        return redirect(request.url)
+
+    file = request.files['file']
+
+    file_name = './testWalks/' + download_id + '.gpx'
```

Comment:
This folder does usually not exist. Perhaps we should create it as well? Or use a different one.